aspnetcore/fundamentals/hosting.md

```diff
@@ -173,7 +180,8 @@ Sets the app's environment.
 **Key**: environment  
 **Type**: *string*  
 **Default**: Production  
-**Set using**: `UseEnvironment`
+**Set using**: `UseEnvironment`  
+**Environment variable**: `ASPNETCORE_ENVIRONMENT`
 
 You can set the *Environment* to any value. Framework-defined values include `Development`, `Staging`, and `Production`. Values aren't case sensitive. By default, the *Environment* is read from the `ASPNETCORE_ENVIRONMENT` environment variable. When using [Visual Studio](https://www.visualstudio.com/), environment variables may be set in the *launchSettings.json* file. For more information, see [Working with Multiple Environments](xref:fundamentals/environments).
```

Comment:
Note that when I make my pass on the next PR, I'll deal with these little probs (e.g., "You").